Readme.md

```diff
@@ -0,0 +1,56 @@
+Answers to assignment 1 of week 2
+---------------------------------
+
+a) The two size classes in iOS are classes for the horizontal and vertical size of devices. In portrait mode it is .regular vertically and .compact horizontally, while both are .compact in landscape mode.
+
+b) Continuous Learning means to always going on in learning and never be finished. This is important in mobile dev since the base technologies evolve (eg Swift in version 5.9 has many new features), new technologies enter the scene (SwiftUI, Combine, SwiftData) and there are many frameworks and tools around which also evolve (CI/CD, DevOps, Cloud).
+
+c) To find the modifiers of a particular view in the UI: Look in the source code. I remember from the videos, that one can set the canvas in Xcode to selectable and then select elements of the UI in the canvas and find, where they are in the code editor, but I cannot find info about this, and if I try this out, it does not work with simple clicking, cmd-clicking, alt-clicking or ctrl-clicking. So I'm clueless here.
+
+d) A breakpoint is a line in the source code, where a debugger will stop execution. Then it is posssible to inspect variables, evaluate expressions ... and then go on in different ways (continue program execution, step over, step into, step out).
+
+e) Access environment values with the property wrapper @Environment, eg @Environment(\.verticalSizeClass) var verticalSizeClass
+
+f) @Environment(\.colorScheme) var colorScheme and then compare it with .dark or .light
+
+g) Magic numbers make it more difficult to evolve an app, it is difficult to see eg if a number 10 should be changed to 12 which 10s have to be edited. Better: Define named constants in an enum
+
+h) To view an app in light and dark mode simultaneously: Define two previews 
+ContentView()
+      .preferredColorScheme(.light)
+ContentView()
+      .preferredColorScheme(.dark)
+
+i) The modifier padding was set after the modifier background.
+
+j) Button("Delete").padding(.horizontal, 10)
+
```

Comment:
You can also pass an array of edges so `.padding([.leading, .trailing] ,10)`. Either way is fine.